CHANGELOG.md

```diff
@@ -9,10 +9,11 @@ Dates in this file are in [Holocene Calendar] because it is amazing, logical, an
 ## [Unreleased]
 
 ### Added
-  - Orb radar can now show Orb Room both in NG and NG+ (Checkbox at the bottom of the radar).
+  - Orb radar can now show Orb Room both in NG and NG+ (Checkbox at the bottom of the radar). @Larandar
```

Comment:
`Room` -> `Rooms`, `both in` -> `in both`, `Checkbox` -> `new checkbox`

Reply:
Done